### DISCUSSION TITLE
Podcast Quality in other languages

### DISCUSSION THREAD

**Discussion by rmadrazo97** [lfnovo/open-notebook#30] [General]
I tried generating a podcast in Spanish, but it's horrible... 
Is there a way to generate better quality / native speech as in English?

thanks!

**Comment by lfnovo** [comment]
What was the audio provider/model you used for the Spanish podcast? OpenAI/Google/Elevanlabs? 
Can you share the file with me? What do you mean by horrible? Was it the content, the audio quality?

**Reply by rmadrazo97** [reply]
Thanks for your answer. 
I'll share the audio file, 
and the model config:

![screencapture-localhost-8080-Podcasts-2024-12-12-19_32_13](https://github.com/user-attachments/assets/1f973d3d-edef-48fe-81b6-4f0e49266473)
[[](url)](https://drive.google.com/file/d/1X1f0wWeAu_4Fj7uFZBxjZKyrpkYUO7Xl/view?usp=drive_link)

**Reply by lfnovo** [reply]
Just requested access to the audio file. Yeah, but I can imagine as I have a similar issue in portuguese. OpenAI is not the best provider for this. Have you tried Gemini or Elevenlabs? For my use cases they work MUCH MUCH better in non-english languages. Eleven Labs is really my favorite. I think you'd like if you try.

**Comment by lfnovo** [comment]
Open Notebook now supports selecting multiple podcast languages, and the podcast stack has changed significantly since this discussion was opened. We are closing this old support thread as outdated.

If Spanish output is still poor on the current release, please open a reproducible bug through the [issue form](https://github.com/lfnovo/open-notebook/issues/new/choose) and include the TTS provider, model, selected language or locale, Open Notebook version, and a short description or sample of the failure. That will give us enough information to diagnose it. Thank you for reporting the original problem.
